controller/appcontroller.go

```diff
@@ -943,6 +944,11 @@ func (ctrl *ApplicationController) watchSettings(ctx context.Context) {
 				ctrl.stateCache.Invalidate()
 				prevResourceExclusions = newSettings.ResourceExclusions
 			}
+			if !reflect.DeepEqual(prevResourceInclusions, newSettings.ResourceInclusions) {
+				log.Infof("resource inclusions modified")
```

Comment:
Mino:but I feel like I'd. want to know what's changed:

```golang
log.WithFields(log.Fields{"prevResourceInclusions": prevResourceInclusions, "newResourceInclusions": newSettings.ResourceInclusions}).Info("resource inclusions modified")
```

Reply:
Added!